ssp

```diff
@@ -1150,35 +1125,54 @@ sub get_tiers_file {    #TODO: Get rid of this in favor of get_tiers_json_href i
     return _http_get( Host => 'httpupdate.cpanel.net', Path => '/cpanelsync/TIERS' );
 }
 
-sub get_process_list {    # Usage of this needs to be deprecated in favor of the %process hash
-    return split /\n/, timed_run( 0, 'ps', 'axwwwf', '-o', 'user,pid,cmd' );
+sub get_process_pid_href {
```

Comment:
You may be able to get something a bit more 'clean' here by using Proc::ProcessTable, though if I recall correctly, that module only recently stopped being 'abandoned' on CPAN.
Still may be worth looking into.

Reply:
I would loooove to use various third-party modules in SSP, even just some under /usr/local/cpanel, but SSP is going to have a very difficult time using any non-core Perl modules and has to jump through hoops to use any non-core modules without entirely failing on some systems (i.e. the recently added JSON stuff).  The only "safe" guarantee is that core Perl is installed on any given system (and sometimes an old version, at that), nothing more.

Reply:
Would be awesome to lean on Proc::ProcessTable when we can ( assuming we implement this into the actual codebase int he future )